### ISSUE TITLE
Social handles take incoorect username.

### ISSUE THREAD

**Issue by Nishtha1209** [rahuldkjain/github-profile-readme-generator#665]
Uppercase letters changed to smaller case while feeding user handles
When I try to type my username for my linkedIn and github handles which start from an uppercase letter. It automatically changes to lowercasem which in turn doesn't link my profiles.

**To Reproduce**

1. Go to 'social'
2. Click on 'github username'
3. Enter a username starting with uppercase, let suppose mu username, "Nishtha1209".
4. It will change them to smaller case(as it might had been made default behavior in the code).

**Expected behavior**
The fields should allow to input exact username

**Screenshots**

**Desktop (please complete the following information):**

- OS: Windows 11
- Browser[chrome]

Images to support:
![Screenshot (89)](https://user-images.githubusercontent.com/84739910/203943132-a5be4cb9-1c65-43be-8110-d1103f63c0b6.png)

**Comment by dk3775** [comment]
Yes, even I have experienced this issue. I think I can fix this and get a PR up!
